crates/sui-core/src/authority/authority_store_pruner.rs

```diff
@@ -252,7 +252,7 @@ impl AuthorityStorePruner {
         );
         let tick_duration = Duration::from_secs(config.pruning_run_delay_seconds.unwrap_or(
             if config.num_epochs_to_retain > 0 {
-                min(1000 * epoch_duration_ms / 2, 60 * 60)
```

Comment:
was this just wrong before?

Reply:
yeah but it didn't cause an issue because the min value would be 60* 60 i.e. 1 hour which is want we wanted it to run with